update/ansible/playbook/tasks/update.yml

```diff
@@ -438,6 +438,21 @@
       register: reread_init__result
       changed_when: "'No config updates to processes' not in reread_init__result.stdout"
 
+    # restarting pmm-managed to regenerate /etc/alertmanager.yml
+    - name: Restart pmm-managed EL7
+      when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
+      command: supervisorctl {{ item }} pmm-managed
+      become: true
+      changed_when: true
+      with_items: ["stop", "remove", "add"]
```

Comment:
Why it's `stop-remove-add`, not just `restart`?

Reply:
This was due to a bug in supervisor v3, now it's for consistency sake I guess.

Reply:
Ok, let's keep it as is then and address it later.

Reply:
Yes, it's for consistency :)

Reply:
don't touch whats working :stuck_out_tongue: